docs/development/index.rst

```diff
@@ -416,3 +415,28 @@ A lot of code that's not gamma-ray specific belongs in other packages
 (e.g. `Scipy`_, `Astropy`_, other Astropy-affiliated packages, `Sherpa`_).
 We currently have quite a bit of code that should be moved "upstream" or already has been,
 but the Gammapy code hasn't been adapted yet.
+
+Assert convention
+-----------------
+
+When performing tests, the preferred numerical assert method is
+`numpy.testing.assert_allclose`. Use
+
+``from numpy.testing import assert_allclose``
+
+at the top of the file and then just use ``assert_allclose`` for
+the tests. This makes the lines shorter, i.e. there is more space
+for the arguments.
+
+``assert_allclose`` covers all use cases for numerical asserts, so
+it should be used consistently everywhere instead of using the
+dozens of other available asserts from pytest or numpy in various
+places.
+
+More details in `numpy.testing.assert_allclose`.
```

Comment:
Remove that line (that's a given that the numpy docs contain more info, that's why you link to it above).